pkg/addons/default/kube_proxy.go

```diff
@@ -55,6 +57,12 @@ func UpdateKubeProxyImageTag(clientSet kubernetes.Interface, controlPlaneVersion
 		}
 		return false, errors.Wrapf(err, "getting %q", KubeProxy)
 	}
+
+	hasArm64NodeSelector := hasArm64NodeSelector(d)
+	if hasArm64NodeSelector {
+		logger.Info("missing arm64 nodeSelector value")
+	}
```

Comment:
Either this is confusing or it's wrong. Should it be `if !hasArm64NodeSelector` then log "missing"?